src/PgNamed.hs

```diff
@@ -51,7 +51,6 @@ import Data.Bifunctor (bimap)
 import Data.ByteString (ByteString)
 import Data.Char (isAlphaNum)
 import Data.Int (Int64)
-import Data.List (lookup)
```

Comment:
I think this will cause failure on older GHCs. I think we should either remote option for failing on unused imports or add `CPP` macro detecting the version of base.

Reply:
It works for the 3 versions of GHC that we explictly say that we support. Older versions than that can't use this package because of base version bounds anyway.